department-of-veterans-affairs/va.gov-cms
# VACMS-5110: Add destination parameter to dash links.

## Description
See #5110

## Testing done
Visual

## QA steps
- As an administrator, go to `section/veterans-health-administration/vet-centers/escanaba-vet-center` and click any link that is not a view.
- Save the node where clicking the link above directed you.
- Confirm that after save, you are redirected to the Escanaba dashboard.

### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
